src/steps.tsx

```diff
@@ -327,8 +339,13 @@ const riskAssessment: Step[] = [
   },
   {
     id: 'askHasChronicLungDiseaseInfo',
-    message:
-      'Not sure? Answer yes if any of the following are true: You have a history of asthma; You have a history of chronic obstructive pulmonary disease/copd or emphysema; You have a history of chronic bronchitis; You have a history of interstitial lung disease; You have a history of a chronic lung disease not listed above.',
+    message: `
+    Not sure? Answer yes if **any** of the following are true: 
```

Comment:
```suggestion
    Not sure? Answer yes if you have a history of**any** of the following: 
```

Then you can remove `You have a history of...` from every bullet points :)